packages/entando-apimanager/README.md

```diff
@@ -73,15 +78,15 @@ This two actions have to be used when logging and logging out a user.
 
 On top of that if any request returns either 401 or 403 status codes the package will logout the user and redirect him to the login page.
 
-### loginUser(username, token)
+### loginUser(username, token, refresh_token=optional)
```

Comment:
it is missing the explanation of the refresh_token parameter, which on top of that is not following our `camelCase` convention for var names

Reply:
noted. Sorry about it. I will elaborate